src/planscape/users/forms.py

```diff
@@ -16,26 +16,19 @@ def save(self, request, **kwargs):
         configuration set via Django's configuration.
 
         """
-        domain = settings.PASSWORD_RESET_DOMAIN
```

Comment:
We should probably remove the PASSWORD_RESET_DOMAIN variable in the Django settings since it's no longer necessary.

Reply:
Done. Thank you